packages/@stylexjs/babel-plugin/__tests__/transform-stylex-createTheme-test.js

```diff
@@ -167,6 +203,30 @@ describe('@stylexjs/babel-plugin', () => {
       expect(metadata).toMatchInlineSnapshot(`
         {
           "stylex": [
+            [
+              "xop34xu",
```

Comment:
This specific hash / class name - `xop34xu` - appears in most of these snapshots because we frequently use `/stylex/packages/vars.stylex.js` as the default file name